<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="lcdhour">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdminute">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdsecond">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>200</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startTime">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>497</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endTime">
    <property name="geometry">
     <rect>
      <x>672</x>
      <y>497</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>结束</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
